debian/control

```diff
@@ -138,6 +138,7 @@ Depends: ivozprovider-profile-common (=${binary:Version}),
         redis-tools,
         netcat-openbsd,
         ssl-cert,
+        libdbd-mariadb-perl,
```

Comment:
If you go ahead with the recomended changes, replace this with libdbd-mysql-perl.
We don't really use mariadb, neither in standalone nor in distributed installs.